### ISSUE TITLE
Exponential notation generates a syntax error.

### ISSUE THREAD

**Issue by bcauchon** [davecparker/code12#130]
```
import Code12.*;
public class TestProgram extends Code12Program
{
	public static void main(String[] args)
	{
		Code12.run(new TestProgram());
	}

	public void start()
	{
		double c = 2.99792458E8;
	}
}
```

**Comment by davecparker** [comment]
I'll give this one to you. I'll show you where in the lexer it needs to go.

**Comment by bcauchon** [comment]
Fixed in [afe72e5](https://github.com/davecparker/code12/commit/afe72e5e04630032af54ab25318ca1fe842f723a)

**Comment by davecparker** [comment]
Verified in UserCode.java locally on my branch.
